docs/intro/zero2hero/html-content.mdx

```diff
@@ -3,3 +3,8 @@ import AsciiDocContent from "/src/components/AsciiDocContent";
 # Rendering HTML content
 
 <AsciiDocContent content="fooo" />
+
```

Comment:
You checked this in by accident. I think we will delete this page in the "base" PR. So can you please revert this. I will go check as I may have _also_ checked this in by accident.

Reply:
✅ Pulled base branch.